internal/cortex/frontend/transport/utils/utils_test.go

```diff
@@ -34,19 +34,16 @@ func verifyMetricCount(t *testing.T, reg *prometheus.Registry, expectedCount int
 
 func TestNewFailedQueryCache(t *testing.T) {
 	reg := prometheus.NewRegistry()
-	cache, err := NewFailedQueryCache(2, reg)
+	cache := NewFailedQueryCache(2, 0, reg)
```

Comment:
Can you add test cases where the TTL is not 0?

Reply:
What does 0 TTL mean? Never expire?

Reply:
0 means no ttl